Makefile

```diff
@@ -180,6 +184,7 @@ help: FORCE
 	@printf "  \e[36mrun-modernize\e[0m                Install and run modernize. Installing is used in CI, but you should probably install modernize using your package manager.\n"
 	@printf "  \e[36mbuild/cover.out\e[0m              Run tests and generate coverage report.\n"
 	@printf "  \e[36mbuild/cover.html\e[0m             Generate an HTML file with source code annotations from the coverage report.\n"
+	@printf "  \e[36m__static-check\e[0m               Run static code checks (internal option to enforce --keep-going)\n"
```

Comment:
This should not be documented in `make help`. Can we make the generator for this part skip targets with prefix `__`?